src/hazelcore/UsageAnalysis.re

```diff
@@ -48,7 +50,7 @@ and find_uses_line = (~steps, x: Var.t, line: UHExp.line): (uses_list, bool) =>
   | CommentLine(_) => ([], false)
   | ExpLine(opseq) => (find_uses_opseq(~steps, x, opseq), false)
   | EmptyLine => ([], false)
-  | LetLine(p, _, def) => (
+  | LetLine(p, def) => (
```

Comment:
regression from dev: usage finding isn't working for the recursive variable

Reply:
Pretty sure this is fixed now but I'm not 100% sure, let's remember to check tmrw